django/http/response.py

```diff
@@ -198,6 +198,8 @@ def set_cookie(self, key, value='', max_age=None, expires=None, path='/',
             self.cookies[key]['secure'] = True
         if httponly:
             self.cookies[key]['httponly'] = True
+        if samesite:
+            self.cookies[key]['samesite'] = samesite.capitalize()
```

Comment:
Why not just expect the setting to be the correct capitalization, `'Lax'` or `'Strict'`? Doing `capitalize()` feels like unnecessary post-processing to me. Is it a matter of setting aesthetics?